enterprise/web.core/src/org/netbeans/modules/web/core/jsploader/JspDataObject.java

```diff
@@ -385,7 +371,7 @@ private void checkRefreshServlet() {
         }
         
         // editor
-        if ((oldServlet == null)/*&&(servletDataObject != null)*/) {
+        if ((oldServlet == null)) {
         } else {
```

Comment:
At least the double bracing should be removed.

```suggestion
        if (oldServlet != null) {
```

Reply:
Applied.